py3status/modules/window_list.py

```diff
@@ -0,0 +1,228 @@
+# -*- coding: utf-8 -*-
+"""
+Display window list asynchronously.
+
+Configuration parameters:
+    button_focus: mouse button to focus a window (default 1)
+    button_next: mouse button to focus next window (default None)
+    button_previous: mouse button to focus previous window (default None)
+    format: display format for this module (default '{format_window}')
+    format_separator: show separator only if more than one (default ' ')
+    format_window: display format for windows
+        (default '\?max_length=30 [\?if=is_urgent&color=urgent \u2756'
+        |[\?if=is_focused&color=focused \u2756|\?color=unfocused \u2756]] {name}')
+
+Format placeholder:
+    {format_window}  format for windows
+    {floating}       number of floating windows      eg 3
+    {scratchpad}     number of scratchpad windows    eg 4
+    {urgent}         number of urgent windows        eg 2
+    {window}         number of windows               eg 11
+
+format_window placeholder:
+    {name}           window title name
+    is_floating      a boolean based on window data
+    is_focused       a boolean based on window data
+    is_scratchpad    a boolean based on window data
+    is_urgent        a boolean based on window data
```

Comment:
This is explicitly boolean (and not placeholders). See format's placeholders. They don't have `is_` because it can be used still... to show numbers. This is only good for controlling how things should appear... so I'd like to make them more clear... and is helpful in organizing (eg, they all stay together because of `is_`...). Same as `gcal` and some other modules.